src/OSPSuite.Core/Serialization/PKMLVersion.cs

```diff
@@ -6,8 +6,8 @@ public static class PKMLVersion
       public const int V5_4_1 = 5;
       public const int V5_5_1 = 6;
       public const int V5_6_1 = 7;
-      public const int NON_CONVERTABLE_VERSION= V5_6_1;
       public const int V6_0_1 = 8;
```

Comment:
@Yuri05 I have removed this version from supported version. This is the only place where we were doing something with the BuildCOnfiguraiton in core and this would become a nightmare to try to support it. Adios